css-pseudo-classes/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <h2>Hover Button</h2>
+    <button class="button">I'm a button!</button>
+    <h2>Striped Table</h2>
+
+    <table>
+      <thead class="thead">
+        <tr class="top">
+          <td>Student Name</td>
+          <td>Course</td>
+          <td>Grade</td>
+        </tr>
+      </thead>
+      <tr>
+        <td>Ada Lovelace</td>
+        <td>Web Developement</td>
+        <td>100</td>
+      </tr>
+      <tr>
+        <td>Bob Martin</td>
+        <td>Web Developement</td>
+        <td>100</td>
+      </tr>
+      <tr>
+        <td>Ryan Dahl</td>
+        <td>JavaScript</td>
+        <td>110</td>
+      </tr>
+      <tr>
+        <td>Kent Beck</td>
+        <td>Cascading Style Sheets</td>
+        <td>90</td>
+      </tr>
+      <tr>
+        <td>Kyle Simpson</td>
+        <td>JavaScript</td>
+        <td>99</td>
+      </tr>
```

Comment:
This should be inside a `tbody`

Reply:
![Screenshot 2023-07-22 at 10 51 01 AM](https://github.com/julieluangpraxay/c0723-code-solutions/assets/130415142/15c0ae76-08fe-4c4a-8f53-04f8b02f6104)